### DISCUSSION TITLE
Reserve static IP according to CN

### DISCUSSION THREAD

**Discussion by alex-airspan** [strongswan/strongswan#1102] [Q&A]
Hi

I want reserve IP's from pool according to uniq CN. What a syntax for that in swanctl file ?

From swanctl file: What a syntax   cn = abcd     ??? 

connections {
   rw {
      local_addrs  = fc74:172:27:123::10
      local {
         auth = rsa
         certs = server-cert.pem
      }
      remote {
         auth = rsa
         cn = ABCD     ??? 
         cacerts = ca.pem		 
      }
   : 
   :
      pools = rw_pool1
}

pools {
   rw_pool1{
      addrs = fc74:172:27:124::/128
   }
}

**Comment by tobiasbrunner** [comment]
If you want to do this via swanctl.conf, you need separate pools and connections for each client. The identity is configured in `remote.id` (you have to configure either the complete subject DN, use wildcards for other RDNs, or configure `charon.rdn_matching` in strongswan.conf). See  [here](https://docs.strongswan.org/docs/5.9/features/vip.html#_responder_configuration) for other options to assign virtual IPs to clients (e.g. via RADIUS or from a database).

**Comment by alex-airspan** [comment]
Hi Tobias 

I tried configure  also with one device -   id = "C=*, O=*, OU=*,  CN=*" , but I got auth fail every time. 
Can you please assist What I missing ?

Thanks,

**Reply by tobiasbrunner** [reply]
You might want to read [this](https://docs.strongswan.org/docs/5.9/config/identityParsing.html) and have a closer look at the logs.
